### PR TITLE
RestKit Release 3.0

### PR DESCRIPTION
**Pull request by mkistler**
See CHANGELOG.md for list of features / fixes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ If your project does not yet have a Podfile, use the `pod init` command in the r
 use_frameworks!
 
 target 'MyApp' do
-    pod 'RestKit', '~> 2.0'
+    pod 'IBMWatsonRestKit', '~> 3.0'
```
Comment (mediumTaj) [inline README.md:43]: Do we still want to call this restkit? Why do we have Watson here? Are we planning to move this to https://github.com/ibm/swift-sdk-core

File: Package.swift [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // swift-tools-version:4.0
 
 /**
- * Copyright IBM Corporation 2016
+ * Copyright IBM Corporation 2016, 2018
```
Comment (mediumTaj) [inline Package.swift:4]: 👍

**Review by mediumTaj — APPROVED** [review]
👍 looks good! Just a couple of comments but I'm not sure if this is something you are planning to do after.

File: README.md (hunk 1 above)
Comment (mkistler) [reply README.md:43]: We are going to move this, but after this major release.

**Pull request merged**